### ISSUE TITLE
[bug] Not found message annotation

### ISSUE THREAD

**Issue by nvlad** [kalessil/yii2inspections#17]
![image](https://cloud.githubusercontent.com/assets/2493052/21939303/353bcda2-d9d0-11e6-832c-afa3cd38d125.png)

with extension "vova07/yii2-imperavi-widget"

**Comment by kalessil** [comment]
Thank you for reporting @nvlad, shouldn't the category be `imperavi`. Or that's a Yii feature for addressing translations in extensions?

**Comment by nvlad** [comment]
Plugin found i18n declaration in extension `vova07/yii2-imperavi-widget` and add in completion list(category & message), also MissingTranslationsInspector mark this message as missed

**Comment by kalessil** [comment]
I see, checking this.

**Comment by TrueXakeP** [comment]
I have this warning too but with my own messages.
Placing messages folder to project root doesn't help.

![warning](https://cloud.githubusercontent.com/assets/5345528/22554109/abc6e5e6-e967-11e6-988c-5d01fbdcb114.png)
common/config.php:
```php
return [
    ...
    'components' => [
        ...
        'i18n'     => [
            'translations' => [
                'app*'      => $appMsgs = [
                    'class'    => 'yii\i18n\PhpMessageSource',
                    'basePath' => '@messages',
                ],
            ],
        ],
    ],
];
```
bootstrap.php:
```php
Yii::setAlias('@messages', dirname(__DIR__) . '/messages');
```

**Comment by kalessil** [comment]
@TrueXakeP: the plugin expect translation files to be directly in ru/uk directory (the common/config.php is not parsed, we are following Yii2/Craft3 directory structure here).

Alternatively in v1.0.0 you can deactivate this specific case reports in the inspection settings (but autocompletion will remain)
